ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly.yaml

```diff
@@ -2708,6 +2708,15 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-vsphere-ipi-remote-worker
+- as: vsphere-ipi-sdn-live-migration-ovn-f28
+  cron: 39 9 1,17 * *
```

Comment:
please update the cron job time by `./ci-operator/config/openshift/openshift-tests-private/tools/generate-cron-entry.sh` tool generate a new one

Reply:
@zhaozhanqi Update it by using above tool.